### ISSUE TITLE
Advice on setting up an environment with WGS84 coordinates system

### ISSUE THREAD

**Issue by alphonsustay** [open-rmf/rmf_traffic_editor#461]
Hi, would like to check what are the requirements needed to setup a traffic-editor environment for using WGS84. For example, getting tiles to display on the editor.

**Comment by Yadunund** [comment]
`Building`->`New`->Enter name of file->`Save`->`Geographical Coordinates`->`Save`

If you reopen the editor with the saved file, it should render OSM tiles in the canvas. If you're seeing yellow tiles instead, there might be a problem with our server 🤔

**Comment by alphonsustay** [comment]
Hey @Yadunund thanks for the prompt response. I can confirm that I have done the recommended steps and I am seeing the yellow tiles. Can I confirm that the server domain is `https://tiles.sandbox.open-rmf.org`

Setup Environment:
Ubuntu20.04 with ROS2 Galactic. This was tested on branches `galactic-devel` and `main`.

**Comment by Yadunund** [comment]
Yeah the server is down so this feature is unfortunately not working at the time. Any fixes will target the new `rmf_site` project and I've opened a ticket there to include this functionality. https://github.com/open-rmf/rmf_site/issues/129
